drivers/volume/portworx/portworx.go

```diff
@@ -194,7 +194,22 @@ func (d *portworx) ValidateCreateVolume(name string, params map[string]string) e
 	}
 
 	// Labels
-	if err := d.schedOps.ValidateAddLabels(vol); err != nil {
+	var pxNodes []api.Node
+	for _, rs := range vol.ReplicaSets {
```

Comment:
are the nodes in multiple replica sets unique? if not, then we will be checking for labels twice on the same nodes.